CHANGES.md

```diff
@@ -1,5 +1,47 @@
 # Changes
 
+## edge-23.4.2
+
+This edge release contains a number of bug fixes.
+
+* CLI
+  * Fixed Helm uninstall issue for HttpRoute
+  * The `linkerd diagnostics policy` command now displays outbound policy when
+    the target resource is a Service
+
+* CNI
+  * Fixed incompatibility issue with AWS CNI addon in EKS, that was
+    forbidding pods to acquire networking after scaling up nodes.
+    (thanks @frimik!)
+  * Added --set flag to install-cni plugin (thanks @amit-62!)
+
+* Control Plane
+  * Fixed an issue where the policy controller always used the default
+    `cluster.local` domain
+  * Send Opaque protocol hint for opaque ports in destination controller
+
+* Helm
+  * Fixed an issue in the viz Helm chart where the namespace metadata template
+    would throw `unexpected argument found` errors
+
+* Multicluster
+  * Remove namespace field from cluster scoped resources to fix pruning
+
+* Proxy
+  * Updated `h2` dependency to include a patch for a theoretical
+    denial-of-service vulnerability discovered in CVE-2023-26964
+  * Bump h2 version to avoid panic issue in v.0.3.17
```

Comment:
I think we can remove this one since the previous item already addresses the h2 upgrade
```suggestion
```